packages/core/src/index.ts

```diff
@@ -2,14 +2,20 @@ export { buildModule } from "./buildModule";
 export * from "./errors";
 export * from "./initialization";
 export { defineModule } from "./new-api/define-module";
+export { Deployer } from "./new-api/deployer";
 /* TODO: how is module constructor getting exposed? */
 export { ModuleConstructor } from "./new-api/internal/module-builder";
+export { FileJournal, MemoryJournal } from "./new-api/journal";
```

Comment:
Same here.

Reply:
Maybe we should clean up the repository and this file, because its getting tricky to understand what should be or not public and why.

(for example, there's a TODO I missed in line 6)

Reply:
Oh, I think the comments I left here should have been submitted in #255

Reply:
I changed the base branch of this pr to reflect that it's based on #255